### PR TITLE
fix(kafka-setup): Make topic name consistent with other images

### PR DESCRIPTION
**Pull request by pedro93**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by david-leifker — APPROVED** [review]

**Review by jjoyce0510 — COMMENTED** [review]
Are these changes backward compatible? Will they break folks with custom topic names?

**Comment by pedro93** [comment]
> Are these changes backward compatible? Will they break folks with custom topic names?

They are not. Folks with custom topic names will have to change the environment variable names they are using to set names. This PR makes that name consistent for all components. Kafka-setup job was the outlier using inconsistent env var names compared to GMS & standalone consumers. I have made a note about this in updating-datahub.md

File: docs/how/updating-datahub.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ This file documents any backwards-incompatible changes in DataHub and assists pe
 
 ### Breaking Changes
 
+- #7103 This should only impact users who have configured explicit non-default names for DataHub's Kafka topics. The environment variables used to configure Kafka topics for DataHub used in the `kafka-setup` docker image have been updated to be in-line with other DataHub components, for more info see our docs on [Configuring Kafka in DataHub
+](https://datahubproject.io/docs/how/kafka-config). They have been suffixed with `_TOPIC`. This change should not affect any user who is using default Kafka names.
```
Comment (jjoyce0510) [inline docs/how/updating-datahub.md:10]: they have been suffixed with _TOPIC_NAME right?

**Review by jjoyce0510 — APPROVED** [review]
Once the doc is updated LGTM

File: docs/how/updating-datahub.md (hunk 1 above)
Comment (pedro93) [reply docs/how/updating-datahub.md]: Updated the doc.

**Pull request merged**